citation/export_data.py

```diff
@@ -1,5 +1,8 @@
 from citation.models import Publication, Platform, Sponsor
 
+ALL_PLATFORMS = Platform.objects.all().values_list("name").order_by("name")
```

Comment:
These static initializations at module import time are problematic for a few reasons. However, the main one is that they won't incorporate updates to Platform or Sponsor.